addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection.scss

```diff
@@ -1,18 +0,0 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_DiscussMobileMailboxSelection {
-    display: flex;
-    flex: 0 0 auto;
-}
-
-.o_DiscussMobileMailboxSelection_button {
-    flex: 1 1 0;
-    padding: map-get($spacers, 2);
```

Comment:
This `padding` is missing with change (left: before, right: after)
<img width="1149" alt="Screenshot 2022-04-15 at 16 28 36" src="https://user-images.githubusercontent.com/6569390/163583006-4778a98b-fe1f-45c6-8b18-2b387186a4c5.png">

Reply:
Indeed. I just fixed it. Thanks! 👍